docs/reference/search/explain.asciidoc

```diff
@@ -1,12 +1,9 @@
 [[search-explain]]
 == Explain API
 
-The explain api computes a score explanation for a query and a specific
-document. This can give useful feedback whether a document matches or
-didn't match a specific query.
+The explain api computes a score explanation for a query and a specific document, letting you know whether a document matches or doesn't match a specific query.
 
-The `index` and `type` parameters expect a single index and a single
-type respectively.
+The `index` and `type` parameters expect a single index and a single type respectively.
 
```

Comment:
I find it misleading to refer to index and type as parameters, but have a section called "All Parameters" that doesn't mention them (because they aren't query string parameters, they're built into the URL). My preference would be to write around it, something like "You must run explain against a particular document. You cannot specify multiple indices, types, or documents." (I don't have the stack running to verify, but I'm pretty sure you can't specify multiple docs...)